src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -57,7 +56,7 @@ public PackageManager() {
     /**
      * Sets up local storage for working with game packages and cache files.
      *
-     * @param gameDirectory directory path for storing game packages
+     * @param gameDirectory  directory path for storing game packages
```

Comment:
Is it common practice to leave multiple whitespaces between the parameter name and description?

If not:
```suggestion
     * @param gameDirectory directory path for storing game packages
```

Reply:
I think the auto-formatting aligned the descriptions here:
```java
* @param gameDirectory  directory path for ...
* @param cacheDirectory directory path for ...
```

Reply:
ah I see